new/analyse/widen-dependency-ranges.ts

```diff
@@ -0,0 +1,32 @@
+import type { Module } from '../types';
+
+import { clone } from 'lodash';
+
+import { logger } from '../helpers/logger';
+
+const { log } = logger('[analyse] wide dependnecy ranges');
+
+// Matches either "^", "~" or "*" at the beginning of the string
+const LOOSE_RANGE_PATTERN = /^[~^*]/;
+
+export const widenDependencyRanges: Module = (env) => {
+  const packagesClone = clone(env.packages);
+
+  log('Analysing packages');
+  let widenCounter = 0;
+  Object.values(packagesClone!).forEach((packageValue) => {
+    // Widen dependencies ranges
+    packageValue.dependsOn.forEach((dependency) => {
+      // except they're already widened
+      if (LOOSE_RANGE_PATTERN.test(dependency.range)) {
+        return;
+      }
+      widenCounter++;
```

Comment:
What is also missing are the parts, we had a chat about:
- You need to add a message under `relatedMessages`. It should be of type patch (i.e. `fix`).
- You need to mark the package, so it's considered changed by setting `determinedIncrementLevel ` to `0`